src/main/java/org/example/App.java

```diff
@@ -15,7 +15,16 @@ public static void main( String[] args ) {
      * @param str Array of words which must be sorted
      * @return List of sorted words
      */
-    public static List<String> sortList(String[] str){
-        return Arrays.asList(str).stream().sorted().collect(Collectors.toList());
+    public static List<String> sortList(String[] str) throws IllegalArgumentException{
```

Comment:
From maintainability point of view, it's better to split validation/sorting/printing logic.

What does it mean?


```
public static void main( String[] args ){
 try {
   validate(args); // validate params in this method, throw exception in this method if necessary.
   List<String> sortedList = sortList(args); // sort list in this method and return a list
   printList(sortedList); // print passed list in this method.
 } catch (RuntimeExeption e) {
  // caught any exception here and print a meaningful message for the user
}
}
```
by doing all mentioned above you will be able in the future to modify the logic in the given method without the necessity to refactor all the app.